admin-views/tribe-options-display.php

```diff
@@ -10,6 +10,18 @@
 	$template_options[$templates[$template]] = $template;
 }
 
+/**
+ * Filter the array of views that are registered for the tribe bar
+ * @param array array() {
+ *     Associative array of views
+ *
```

Comment:
Wondering, should it be made clearer that this is an array of arrays (only the inner arrays are associative)?